### PR TITLE
Added show image

### PR DESCRIPTION
**Pull request by rodrigopavezi**
Added tests for Atom

@stulees please review.

Presentation here:

http://preview.risevision.com/Viewer.html?type=presentation&id=b6680ac9-0345-4bb0-bd7a-1cd5a73e4845

### CODE REVIEW

File: src/widget/content.js [hunk 1]
```diff
@@ -97,9 +99,29 @@ RiseVision.RSS.Content = function (prefs, params) {
     return author;
   }
 
+  function _getImageUrl(item) {
+    var imageUrl = null;
+
+    // RSS 2.0
+    if (_.has(item, "media:content")) {
```
Comment (stulees) [inline src/widget/content.js]: @rodrigopavezi This is probably okay for now, but we will likely need to accommodate multiple `<media:content>` elements being contained within a `<media:group>` element. In that scenario, we would probably just take the first one in the group.

**Comment by stulees** [comment]
@rodrigopavezi One comment regarding potential `<media:group>` elements, otherwise looks good. I am noticing padding being applied to the image which is making it not flush with the content to the right. Was this intentional?

**Comment by stulees** [comment]
@rodrigopavezi Just noticed this offset between image and content - http://www.screencast.com/t/m0atfTDk7pU

**Comment by rodrigopavezi** [comment]
@stulees made changes, please review. cheers

**Comment by stulees** [comment]
@rodrigopavezi Looks good :+1:

**Pull request merged**